Applications/Tizen_native/CustomShortcut/res/edje/main.edc

```diff
@@ -128,8 +128,8 @@ collections {
           rel2.relative: 1.0 0.8;
         }
       }
-      PART_BUTTON("draw/reset", "↩️", 0.2, 0.8, 0.45, 0.9)
-      PART_BUTTON("draw/proceed", "▶️", 0.55, 0.8, 0.8, 0.9)
+      PART_BUTTON("draw/reset", "↩️", 0.2, 0.83, 0.45, 0.9)
+      PART_BUTTON("draw/proceed", "▶️", 0.55, 0.83, 0.81, 0.9)
```

Comment:
As these values are fixed, does this mean that it is screen size dependent? If yes, please add the device you use to test (watch version 2 or 3 etc) in doc of this application.

Reply:
conformant object should resize according to the screen size, but better clarify tested device. Will do in this PR